vulnerabilities/importers/osv.py

```diff
@@ -113,14 +129,19 @@ def get_severities(raw_data) -> Iterable[VulnerabilitySeverity]:
     """
     Yield VulnerabilitySeverity extracted from a mapping of OSV ``raw_data``
     """
-    for severity in raw_data.get("severity") or []:
-        if severity.get("type") == "CVSS_V3":
-            vector = severity["score"]
-            system = SCORING_SYSTEMS["cvssv3.1"]
-            score = system.compute(vector)
-            yield VulnerabilitySeverity(system=system, value=score, scoring_elements=vector)
-        else:
-            logger.error(f"Unsupported severity type: {severity!r} for OSV id: {raw_data['id']!r}")
+    try:
+        for severity in raw_data.get("severity") or []:
+            if severity.get("type") == "CVSS_V3":
+                vector = severity["score"]
+                system = SCORING_SYSTEMS["cvssv3.1"]
+                score = system.compute(vector)
+                yield VulnerabilitySeverity(system=system, value=score, scoring_elements=vector)
+            else:
+                logger.error(
```

Comment:
Is there such a case? what other values?

Reply:
According to OSV schema `CVSS_V2` but according to logs nothing

Reply:
@pombredanne Should I add support for CVSS_V2 ?

Reply:
@ziadhany Nope :) we are trying to get away from it!